packages/flutter_tools/lib/src/commands/run.dart

```diff
@@ -317,11 +317,13 @@ class RunCommand extends RunCommandBase {
     }
 
     DateTime appStartedTime;
-    // Sync completer so the completing agent attaching to the resident doesn't 
-    // need to know about analytics. 
+    // Sync completer so the completing agent attaching to the resident doesn't
+    // need to know about analytics.
     //
     // Do not add more operations to the future.
     final Completer<Null> appStartedTimeRecorder = new Completer<Null>.sync();
+    // Callback hookups' Futures are uninteresting.
```

Comment:
For this specific kind of future we should have a string we can easily search for, something like:

```dart
  appStartedTimeRecorder.future.then( // ignore: unawaited_futures (this "then" callback can't throw)
```